app/(dashboard)/instances/components/InstanceForm.tsx

```
@@ -0,0 +1,615 @@
+
+          if (
+            value === undefined ||
+            (typeof value === "string" && !value.trim())
+          ) {
+            delete data.requestParams[key];
+          }
+        }
+
+        // Remove cloud_provider_native_network_id if cloudProvider is GCP
+        if (data.cloudProvider === "GCP") {
+          delete data.requestParams.cloud_provider_native_network_id;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)